.github/workflows/on-issue-creation.yml

```diff
@@ -36,33 +37,3 @@ jobs:
           issue-number: ${{ github.event.issue.number }}
           body: |
             The current version of Grafana OnCall, at the time this issue was opened, is ${{ steps.get-latest-tag.outputs.tag }}. If your issue pertains to an older version of Grafana OnCall, please be sure to list it in the PR description. Thank you :smile:!
-
-  verify-that-atleast-one-label-is-added:
-    name: Verify that atleast one label is added to issue
-    # basically only run this job if the issue has no labels at the time of creation
-    # AND the person creating the issue is a member of the Grafana org
-    # https://docs.github.com/en/graphql/reference/enums#commentauthorassociation
-    # MEMBER - Author is a member of the organization that owns the repository.
-    # OWNER - Author is the owner of the repository.
-    if: join(github.event.issue.labels.*.name, '') == '' && contains(fromJSON('["MEMBER", "OWNER"]'), github.event.issue.author_association)
```

Comment:
this was an experiment. basically it would yell at the issue author, only if they were a Grafana team member, and they didn't put any labels on the issue at creation time. I think we can get rid of this in lieu of the new templates